src/TriggerBinding/SqlTriggerValueProvider.cs

```diff
@@ -24,21 +27,37 @@ internal class SqlTriggerValueProvider : IValueProvider
         /// <param name="tableName">Name of the user table</param>
         public SqlTriggerValueProvider(Type parameterType, object value, string tableName)
         {
-            this.Type = parameterType;
+            this._parameterType = parameterType;
             this._value = value;
             this._tableName = tableName;
+            this._isString = parameterType == typeof(string);
         }
 
         /// <summary>
         /// Gets the trigger argument value.
         /// </summary>
-        public Type Type { get; }
+        public Type Type
```

Comment:
Followed the CosmosDB extension implementation [here](https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.CosmosDB/Trigger/CosmosdbTriggerValueBinder.cs.). They also support JArray but the languages I tested so far (JS, PS, Python) are only using String. We can add JArray later if needed for other languages/scenarios.